src/sql/workbench/api/common/extHostModelView.ts

```diff
@@ -566,6 +566,22 @@ class ComponentWrapper implements azdata.Component {
 		this.setProperty('ariaLabel', v);
 	}
 
+	public get ariaRole(): string {
+		return this.properties['ariaRole'];
+	}
+
+	public set ariaRole(v: string) {
+		this.setProperty('ariaRole', v);
+	}
+
+	public get ariaSelected(): boolean {
+		return this.properties['ariaSelected'];
+	}
+
+	public set ariaSelected(v: boolean) {
```

Comment:
Just checking, but unlike "focus", selection _doesn't_ have to be somewhere, right? So there's no weird effect to setting this to false?

Reply:
Correct. This is just for screen readers and will having it as false will just mean that they won't read off "selected" when the item is read by the screen reader. Removing the attribute has the same effect.